operator/pkg/reconciliation/constructor.go

```diff
@@ -6,351 +6,18 @@ package reconciliation
 // This file defines constructors for k8s objects
```

Comment:
This file should now contain miscellaneous construction logic for everything that does not have a dedicated construct_foo.go file.